src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -3869,6 +3866,9 @@
   <data name="Argument_EmptyString" xml:space="preserve">
     <value>The value cannot be an empty string.</value>
   </data>
+  <data name="Argument_EmptyOrWhiteSpaceString" xml:space="preserve">
```

Comment:
Is this used anywhere?

Reply:
Nope, it's a holdover from another branch where I was adding ThrowIfNullOrWhiteSpace.